tests/apollo/test_skvbc_preexecution.py

```diff
@@ -191,3 +193,32 @@ async def test_view_change(self, bft_network):
                 break
 
         self.assertEqual(new_last_block, last_block)
+
+    @with_trio
+    @with_bft_network(start_replica_cmd)
+    async def test_f_nonprimary_crash_amid_parallel_tx(self, bft_network):
+        '''
+        Crash f nonprimary replicas and submit X parallel write submissions.
+        Block processing of the network should be unaffected with f-count interruption.
+        Final block length should match submitted transactions count exactly.
+        '''
+        bft_network.start_all_replicas()
+        skvbc = kvbc.SimpleKVBCProtocol(bft_network)
+        
+        num_of_requests = 10
```

Comment:
let's make the test non-trivial by running, say 50 requests

Reply:
Certainly.

Reply:
FYI, the test runs about 50~60 seconds with 50 reqs. (Testing all 4,6,7 networks)